masterdata/configuration/concepts/drugForms.csv

```diff
@@ -2,4 +2,6 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Descript
 9bb0795c-4ff0-0305-1990-000000000034,,,Injection,,,,,,Misc,N/A,,,,,,,,,,,
 9bb0795c-4ff0-0305-1990-000000000035,,,Drops,,,,,,Misc,N/A,,,,,,,,,,,
 9bb0795c-4ff0-0305-1990-000000000036,,,Jelly,,,,,,Misc,N/A,,,,,,,,,,,
-9bb0795c-4ff0-0305-1990-000000000037,,,Pessary,,,,,,Misc,N/A,,,,,,,,,,,
\ No newline at end of file
+9bb0795c-4ff0-0305-1990-000000000037,,,Pessary,,,,,,Misc,N/A,,,,,,,,,,,
+e466569c-1e90-11ef-8cfd-0242ac120003,,,Tablet(s),tablet,,,,,,Misc,N/A,,,,,,,,,,,
```

Comment:
There are existing concept named as Tablet, Capsule. Are we renaming the same or these are new concepts ?

Reply:
Yes, I'm renaming the already existing one by using its same uuid. https://ciel.standard.mybahmni.in/openmrs/dictionary/concept.htm?conceptId=64

Reply:
@SanoferSameera where is this UUID coming from ? Do we have it fixed somewhere in `bahmni-core` ? Can you validate the source of creation of this concept ?

cc. @angshu @gokultw